external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaDataConsumer.scala

```diff
@@ -46,6 +47,13 @@ private[kafka010] class InternalKafkaConsumer(
 
   val groupId = kafkaParams.get(ConsumerConfig.GROUP_ID_CONFIG).asInstanceOf[String]
 
+  private[kafka010] val clusterConfig = KafkaTokenUtil.findMatchingTokenClusterConfig(
+    SparkEnv.get.conf, kafkaParams.get(CommonClientConfigs.BOOTSTRAP_SERVERS_CONFIG)
+      .asInstanceOf[String])
+
+  // Kafka consumer is not able to give back the params instantiated with so we need to store it.
+  // It must be updated all the time when new consumer created.
```

Comment:
Not sure what github is doing with my comments when I re-open the old conversation, so opening a new one.

Looks like the old comment crept back in and needs to be fixed again.

Reply:
Fixed.

BTW, I've also tried to re-open that conversation just to test it and even if I clicked the button not happened anything. Interesting what happens on github nowadays...